estuary/api/v1.py

```diff
@@ -121,6 +121,9 @@ def _get_partial_story(results, reverse=False):
         rv['meta']['requested_node_index'] = 0
         rv['meta']['story_type'] = story_manager.__class__.__name__[:-12].lower()
         rv['meta']['wait_times'] = [0]
+        rv['meta']['total_wait_time'] = [0]
+        rv['meta']['total_processing_time'] = [0]
```

Comment:
Optional: Wouldn't this be available to calculate?

Reply:
What would be passed in for `results`? Since `results` must be nothing in order to get to this part of the code.